pkg/labels/labels.go

```diff
@@ -285,11 +284,19 @@ func Compare(a, b Labels) int {
 	}
 
 	for i := 0; i < l; i++ {
-		if d := strings.Compare(a[i].Name, b[i].Name); d != 0 {
-			return d
+		if a[i].Name != b[i].Name {
+			if a[i].Name < b[i].Name {
+				return -1
+			} else {
```

Comment:
We can drop the else block and just have return outside in both places. Not a blocker.

Reply:
In this case I think it's clearer this way.